server/models/images.js

```diff
@@ -46,3 +46,18 @@ module.exports.deleteEntry = (id, callback) => {
     callback(null, results);
   });
 };
+
+module.exports.updateEntry = (imageToUpdate, callback) => {
+  // const queryStr = 'SELECT * FROM `images` WHERE hostel_id = ?';
+  // const queryArgs = [id];
+  const queryStr = 'UPDATE `images` SET `file_name` = ?, `url` = ?, `description` = ?, `hostel_id` = ? WHERE `id` = ?';
+  const queryArgs = [imageToUpdate.fileName, imageToUpdate.url, imageToUpdate.description, imageToUpdate.hostelId, imageToUpdate.id];
```

Comment:
This setup can be don automatically and not require all of the row entries if you pass an object into a single ? like below. 

'UPDATE `images` SET ? where id=?' -> qArgs [{}, id]

That way you can just update one field without sending all the data. (lmk if I can answer further Q's)